WordPress/Classes/ViewRelated/Post/PostListEditorPresenter.swift

```diff
@@ -8,8 +8,22 @@ struct PostListEditorPresenter {
 
     static func handle(post: Post, in postListViewController: PostListViewController) {
 
-        // Autosaves are ignored for posts with local changes.
-        if !post.hasLocalChanges(), post.hasAutosaveRevision, let saveDate = post.dateModified, let autosaveDate = post.autosaveModifiedDate {
+        if post.hasVersionConflict() {
+            let conflictResolutionAlert = presentConflictResolutionAlert(for: post) { keepLocal in
+                switch keepLocal {
```

Comment:
I'm very curious about that... using a `switch` for a Boolean has any advantage over an `if`?

Reply:
it's just cleaner, in my eyes. I don't have strong feelings about it.